src/extremes_based/extremesdim.jl

```diff
@@ -34,28 +71,21 @@ function extremevaltheory_dims(X, p; kw...)
 end
 
 """
-    extremevaltheory_dims_persistences(x::AbstractStateSpaceSet, p::Real; kwargs...)
+    extremevaltheory_dims_persistences(x::AbstractStateSpaceSet, p::type; kwargs...)
```

Comment:
`type` doesn't exist. It is not a type that actually exists. It is better to write just `p` instead of `p::type`.